templates/terraform/update_encoder/cloudiot_device_registry.go.erb

```diff
@@ -0,0 +1,27 @@
+config := meta.(*Config)
```

Comment:
Didn't look too closely- is this the exact same code as the create encoder? If so, I'd recommend using the same file for both in order to have fewer things that need to be maintained in the future.

Reply:
done